tests/hazmat/backends/test_openssl.py

```diff
@@ -467,6 +467,28 @@ def test_unsupported_oaep_label_decrypt(self):
                 )
             )
 
+    def test_supported_oaep_decrypt(self):
+        private_key = RSA_KEY_512.private_key(backend)
+
+        ciphertext = private_key.public_key().encrypt(
+            b'secure data',
+            padding.OAEP(
+                mgf=padding.MGF1(algorithm=hashes.SHA1()),
+                algorithm=hashes.SHA1(),
+                label=None
+            )
+        )
+
+        with pytest.raises(ValueError):
+            RSA_KEY_512_ALT.private_key(backend).decrypt(
+                ciphertext,
```

Comment:
Can you please move the `RSA_KEY_512_ALT.private_key(backend)` call outside of the `with pytest.raises(ValueError)` block.